docs/guides/cross-referencing-with-sphinx.rst

```diff
@@ -263,7 +360,7 @@ That will be rendered as:
 Invalid targets
 ---------------
 
-If we reference an invalid or undefined target Sphinx will warn us.
+If you reference an invalid or undefined target Sphinx will warn us.
```

Comment:
Guessing we'll do a simple style guide here soon :)

Reply:
Getting closer :)